### ISSUE TITLE
HTTPS not supported

### ISSUE THREAD

**Issue by daviddoria** [cpp-netlib/cpp-netlib#718]
I built cpp-netlib (0.13 branch on github) with 

CPP-NETLIB_ENABLE_HTTPS=ON

but when I try the "hello world" example with:

std::string requestURL = "https://....";

http::client client;
http::client::request request(requestURL);
http::client::response response = client.get(request);

the output is:

"HTTPS not supported."

Is there any way to make this work?

Thanks,

David

**Comment by daviddoria** [comment]
I found on a forum that you can add this preprocessor definition before the client.hpp include:

#define BOOST_NETWORK_ENABLE_HTTPS
#include <boost/network/protocol/http/client.hpp>

and then it works. This should probably be in the "getting started" guides :)

**Comment by anonimal** [comment]
http://cpp-netlib.org/0.12.0/reference/http_client.html#features
https://github.com/cpp-netlib/cpp-netlib/blob/master/CMakeLists.txt#L83

**Comment by deanberris** [comment]
I agree @daviddoria that this ought to be made clearer in the documentation -- do you or @anonimal mind sending in a pull request to make this clearer? The docs are in `lib/network/docs/` last I checked (and they're in ReStructuredText).

**Comment by daviddoria** [comment]
@deanberris I guess as a new user I was hoping to see something about this here http://cpp-netlib.org/0.12.0/index.html (./libs/network/doc/index.rst) as my thought process was "take the Hello World example, change the server to the server I want to query, and dump the data to make sure I see what I expect". As @anonimal pointed out, this is actually very clearly explained, just not right in this first example. I'd actually now say that 
https://github.com/cpp-netlib/cpp-netlib/blob/master/CMakeLists.txt#L83
is the right way to go so that no documentation changes are required at all and everything "just works".
